integration/benchmark/contLoadGenerator.go

```diff
@@ -58,6 +58,8 @@ type ContLoadGenerator struct {
 	accounts            []*flowAccount
 	availableAccounts   chan *flowAccount // queue with accounts available for workers
 	workerStatsTracker  *WorkerStatsTracker
+	mu                  sync.Mutex
```

Comment:
it's impossible to tell what's guarded by this mutex (if not looking at this specific PR).  add comment next to fields that are guarded by this mutex.  some newlines to divide the fields into sections would be useful as well.